rust/crd/src/authentication.rs

```diff
@@ -765,7 +765,6 @@ mod tests {
             error_message
         );
     }
-
```

Comment:
**nit:** The coding style guide suggests an empty line between functions.

Reply:
done https://github.com/stackabletech/airflow-operator/pull/530/commits/b4d0e2fb818c5144a293d0583d4232eef9216661